examples/hacker-news/esy.json

```diff
@@ -16,14 +16,14 @@
   "dependencies": {
     "@esy-ocaml/reason": ">=3.4.0",
     "@brisk/macos": "*",
-    "@opam/dune": "^1.7.2",
+    "@opam/dune": "^1.8.2",
     "@opam/httpkit-lwt": "*",
     "ocaml": "~4.7.1000",
     "graphql_ppx": "briskml/graphql_ppx#ee86d23",
     "@opam/core_kernel": "v0.11.1"
   },
   "devDependencies": {
-    "@opam/merlin": "^3.2.2",
+    "@opam/merlin-lsp": "ocaml/merlin:merlin-lsp.opam#7cccd4f",
```

Comment:
Should I add merlin back?

Reply:
Yes, until we get proper tooling for vscode.